lib/smart_answer_flows/benefit-cap-calculator.rb

```diff
@@ -92,21 +99,52 @@ def define
           if response == "none"
             outcome :outcome_not_affected
           else
-            question BenefitCapCalculatorFlow.next_benefit_amount_question(config.questions, benefit_types)
+            question BenefitCapCalculatorFlow.next_benefit_amount_question(config.questions(chosen_cap), benefit_types)
+          end
+        end
+      end
+
+      #Q4 future flow
+      checkbox_question :receiving_non_exemption_benefits_future? do
+        config.benefits(:future).keys.each do |benefit|
+          option benefit
+        end
+
+        next_node_calculation :benefit_types do |response|
+          response.split(",").map(&:to_sym)
+        end
+
+        precalculate :benefit_options do
+          config.descriptions(chosen_cap).merge(none_above: "None of the above")
+        end
+
+        calculate :total_benefits do
+          0
+        end
+
+        calculate :benefit_cap do
+          0
+        end
```

Comment:
I've just realised that the [Storing data for later use][1] page doesn't say anything about at which point the template is rendered. I've added a [new Trello card][2] to suggest that we add that.

[1]: https://github.com/alphagov/smart-answers/blob/master/doc/storing-data.md
[2]: https://trello.com/c/WxANAohC